site/src/docs/components/header/index.mdx

```diff
@@ -248,6 +248,16 @@ Header.LanguageSelector offers the languages for the whole service. It is recomm
 - See example for Header.LanguageSelector <Link openInNewTab openInNewTabAriaLabel="Opens in a new tab" href="/storybook/react/?path=/story/components-header--with-full-features">in Storybook</Link>.
 - Under large sized screens (992px and down) the Header.LanguageSelector moves under the Header.ActionBar.
 
```

Comment:
Should we describe also the default language selector with somehting like:

Use the Header.LanguageSelector when:
- Responsive mobile view is needed
- Some languages are shown outside the language dropdown with isPrimary: true, dropdown for secondary languages is left out if there are not any other languages
- There is a need for additional languages

Reply:
https://city-of-helsinki.github.io/hds-demo/hds-2027-header-docs/components/header/